### PR TITLE
add new line in alert event. format HTML

### PR DESCRIPTION
**Pull request by heyalexej**
Fixed the new line that you tried to emulate by having plenty of spaces, [see here](http://stackoverflow.com/a/1841458).

Sometimes and for some reason, the white lemon doesn't show up. It's there and linked right but I don't know why it tries to fetch it the way it does. I've seen this happening with your portraits as well. After a few reloads however, it works.

see 404 errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,259 +1,241 @@
...
+                    <ul class="nav navbar-nav">
+                        <!-- Hidden li included to remove active class from about link when scrolled up past about section -->
+                        <li class="hidden">
+                            <a href="#page-top"></a>
+                        </li>
+                        <li>
+                            <a class="page-scroll" href="#about">Team</a>
+                        </li>
+                        <li>
+                            <a class="page-scroll" href="#download">Services</a>
```
Comment (heyalexej) [inline index.html:60]: I would change that to `href="#services"` and of course apply all the changes in the HTML and CSS that go with it.

File: index.html [hunk 2]
```diff
@@ -1,259 +1,241 @@
...
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="">
+        <meta name="author" content="">
+
+        <title>THE SQUEEZYLEMON</title>
```
Comment (heyalexej) [inline index.html:10]: Add a compelling title. Something like "THE SQUEEZYLEMON - Creative Agency Bla Bla". Use this [tool](http://snippetoptimizer.net/) to help you with character limits. Also fill out the description and remove the author tag.

Comment (nilsvanbreen) [inline index.html:10]: Hey Lady,

How are you? We arrived yesterday in the evening, slept on the floor at
some friends and today finally in a nice guesthouse. Next time we take the
plane, what a horrible bus ride (no aircon). Thanks for you messages, we're
going to change it all!

Did you send out the app? Already got some responses?

Cheers!

Nils & Bouke

2015-05-05 16:36 GMT+07:00 Alexej notifications@github.com:

**Comment by heyalexej** [comment]
You can simply accept my changes into that branch with the button.
merge it

then run: `git pull --all` from your command line when you're in the project directory. With `git branch` you can see which branch you're on. If you're not on `master`, switch to master with `git checkout master`.

 Now apply the rest of the changes I proposed. Remember, you're on master now but the published page is the branch `gh-pages`. After you are done with everything, added it, committed it, you can switch to `gh-pages` and merge the changes from the master branch into it with `git merge master`. Push again and you're done.

**Comment by nilsvanbreen** [comment]
I'm not sure if it worked. can you tell?

**Comment by heyalexej** [comment]
As you can see [here](https://github.com/nilsvanbreen/thesqueezylemon.com/commits/gh-pages), everything went exactly as expected. Good job.

**Comment by nilsvanbreen** [comment]
Nice! i just opened the file in sublime and i see the changes :)

**Comment by nilsvanbreen** [comment]
how is the tool doing? got already some useful feedback?

**Pull request merged**